utils/pre-swap-checker/contracts/PreSwapChecker.sol

```diff
@@ -144,14 +163,24 @@ contract PreSwapChecker {
     return (address(tokenRegistry.transferHandlers(kind)) != address(0));
   }
 
-  // checks for valid interfaces for ERC165 tokens, ERC721
+  /**
+    * @notice Checks for valid interfaces for
+    * ERC165 tokens, ERC721
```

Comment:
I don't understand this sentence? `Checks for valid interfaces for ERC165 tokens, ERC721`

Reply:
the hasERC721Interfaces checks both that the token is an ERC165 and ERC721 though i'll change it as agree it's confusing. and a bad copy-paste.